app/controllers/books_controller.rb

```diff
@@ -1,5 +1,24 @@
 class BooksController < ApplicationController
+  include Wor::Paginate
+
+  def index
+    render_paginated Book.all
+  end
+
   def show
+    book = Book.find(params[:id])
+
+    render(
+      json: Panko::Response.create do |r|
+        {
+          success: true,
+          book: r.serializer(book, BookSerializer)
+        }
+      end
+    )
+  end
+
+  def book_information
```

Comment:
you should create an OpenLibraryController and move this method, and create the route